spec/controllers/user_credentials_controller_spec.rb

```diff
@@ -30,6 +31,14 @@ def valid_attributes(options = {})
     end
   end
 
+  describe "Post import" do
```

Comment:
It would also be great to add a test that ensure the redirect works when one of the credentials was not saved. When you run the test with `COVERAGE=true rake spec` you should get a test coverage report in `coverage/index.html` which shows the code paths that are not covered by tests.